python/ray/tune/execution/trial_runner.py

```diff
@@ -1295,7 +1295,7 @@ def _execute_action(self, trial: Trial, decision: str):
         if decision == TrialScheduler.CONTINUE:
             self.trial_executor.continue_training(trial)
         elif decision == TrialScheduler.PAUSE:
-            self.trial_executor.pause_trial(trial)
+            self.pause_trial(trial)
```

Comment:
I made clearing the trial's entry in `_cached_trial_decisions` the default behavior for pausing.

Reply:
sounds good!